docs/community/developer-guide.md

```diff
@@ -32,7 +32,7 @@ The easiest way to start hacking on AKS Engine is to use a Docker-based environm
 $ make dev
 ```
 
-Or on Windows:
+Or on Windows (ensure Docker is configured for Linux containers on Windows):
```

Comment:
or run `& 'C:\Program Files\Docker\Docker\DockerCli.exe' -SwitchLinuxEngine`

Reply:
but that shouldn't be done automatically - if someone has windows containers running they'll lose the ability to manage them